src/Tribe/PUE/Checker.php

```diff
@@ -345,6 +353,8 @@ public function set_key_status( $valid ) {
 			// We set a transient in addition to an option for compatibility reasons.
 			// @todo remove transient in a major feature release where we release all plugins.
 			set_transient( $this->pue_key_status_transient_name, $status, $this->check_period * HOUR_IN_SECONDS );
+
+			delete_transient( self::IS_ANY_LICENSE_VALID_TRANSIENT_KEY );
```

Comment:
Lets save resources - at this point we can say with confidence if we have one valid or not. So if its valid don't delete rather update. If it's invalid delete.

Reply:
I made the change. My original thought process behind always deleting it was to make sure that `is_any_license_valid` is in sync with the licenses. For example, what if `set_key_status` returns false, but we still have `self::IS_ANY_LICENSE_VALID_TRANSIENT_KEY` set to valid?

Reply:
I think we can respond to that by knowing what would go wrong if there are no valid licenses but the transient still says there is one valid.